src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -117,7 +117,7 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
     @observable searchCaseInput:string;
     @observable searchMutationInput:string;
     @observable viewMutationType:boolean = true;
-    @observable viewCopyNumber:boolean = true;
+    @observable viewCopyNumber:boolean = false;
```

Comment:
what's this?

Reply:
make copy number mode false by default

Reply:
https://github.com/cBioPortal/cbioportal/issues/5082